src/ray/raylet/scheduling/cluster_resource_scheduler.h

```diff
@@ -168,6 +168,13 @@ class ClusterResourceScheduler : public ClusterResourceSchedulerInterface {
                               const std::string &resource_name,
                               double resource_total) override;
 
+  /// Re-init the capacity of the local resources.
+  ///
+  /// \param resource_name: Resource which we want to update.
+  /// \param resource_total: New capacity of the resource.
+  void UpdateLocalResourceCapacity(const std::string &resource_name,
```

Comment:
Do we still need this in this case? If the delete events don’t remove local resources, we don’t need to re-init, or am I missing sth here?

Reply:
yeah, you are right, we can remove the previous code safely now.